ci-operator/jobs/openshift/cluster-update-keys/openshift-cluster-update-keys-master-presubmits.yaml

```diff
@@ -172,3 +172,28 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: api.ci
+    context: ci/prow/make
+    decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-master-make
+    rerun_command: /test make
+    spec:
+      containers:
+      - args:
+        - -c
+        - make ci rhel && git diff --exit-code
+        command:
+        - sh
+        image: cli
```

Comment:
[`description: Pod pending timeout.`][1], so this must not be the way to reference the local copy of the `cli` image.  Not sure what I should be using instead...

[1]: https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/10064/rehearse-10064-pull-ci-openshift-installer-master-make/1280202576947580928

Reply:
try `pipeline:cli`?

Reply:
`ci-operator:latest` seems like it would be the `ci-operator` image.  Not sure if they have `oc` installed?  I don't think we need to involve `ci-operator` at all here, unless that's the only way to set up a local `cli` image.  I've pushed 9d8af0035b -> f90a3aef3d, pivoting to an explicit `registry.svc.ci.openshift.org/ocp/4.6`, although I'm not exited about that going stale during job forking/backporting either.

Reply:
Hmm, looks like [no `make` in the CLI image][1].  @smarterclayton , how much do you care about presubmits for this?  Is it worth adding a `Dockerfile` to `cluster-update-keys` to build an image with both `oc` and `make` (and whatever else we need, e.g. `gcp`)?

[1]: https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/10064/rehearse-10064-pull-ci-openshift-cluster-update-keys-master-make/1280248704007147520

Reply:
you have to reference cli in your base images, like you reference base.